packages/block-library/src/social-link/edit.js

```diff
@@ -79,6 +79,7 @@ const SocialLinkEdit = ( {
 	const ref = useRef();
 	const IconComponent = getIconBySite( service );
 	const socialLinkName = getNameBySite( service );
+	const socialLinkLabel = label ? label : socialLinkName;
```

Comment:
```suggestion
	const socialLinkLabel = label ?? socialLinkName;
```

**Nit (non-blocking):** We can use nullish coalescing operator here.